documents/anchor_task.go

```diff
@@ -32,10 +32,10 @@ type documentAnchorTask struct {
 	transactions.BaseTask
 
 	id       []byte
-	tenantID common.Address
+	tenantID identity.CentID
 
 	// state
-	config        config.Configuration
+	config        configstore.Service
```

Comment:
For asynchronous tasks I rather pass static values instead of loading them dynamically when the message reaches the consumer. It adds up more predictability to the queueing/dequeuing flow.